components/brave_wallet_ui/page/screens/swap/hooks/useJupiter.ts

```diff
@@ -138,7 +139,7 @@ export function useJupiter(params: SwapParams) {
           outputToken:
             overriddenParams.toToken.contractAddress ||
             WRAPPED_SOL_CONTRACT_ADDRESS,
-          taker: overriddenParams.fromAddress
+          taker: overriddenParams.fromAccountId.address
```

Comment:
Let's get address from AccountInfo. Address field in AccountId was/is a temporary measure and to be removed soon™
```
const fromAccount = useAccountQuery(fromAccountId)
fromAccount.address
```

Reply:
Fixed [here](https://github.com/brave/brave-core/pull/21406/commits/a35fa631856a6c383bea9b9221efbd5e088d1fb4), but without the `useAccountQuery()` hook.